gallia-prehistoire.csl

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" delimiter-precedes-et-al="never" et-al-use-first="1" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
+  <info>
+    <title>Gallia Préhistoire (French)</title>
+    <title-short>GP</title-short>
+    <id>http://www.zotero.org/styles/gallia-prehistoire</id>
+    <link href="http://www.zotero.org/styles/gallia-prehistoire" rel="self"/>
+    <link href="http://www.persee.fr/collection/galip" rel="documentation"/>
```

Comment:
@tomax31, did you base this on an existing CSL style? If so, we'd like to have a template link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link).